src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,46 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: {
+    user: {
+      id: number;
+      name: string;
+      username: string;
+      email: string;
+    } | null;
+    id: number;
+    title: string,
+    completed: boolean;
+    userId: number;
+  }
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  if (!todo) {
```

Comment:
Although your function will return null if 'todo' is not provided, it would be better to set a default value for 'todo' in the function parameters to prevent any unexpected errors.